renaming_aps/README.md

```diff
@@ -0,0 +1,78 @@
+# Renaming APs
+This is a Python script that uses the [Pycentral](https://pypi.org/project/pycentral/) library to achieve the following steps on an Aruba Central account- 
+1. Copy device configuration
+2. Change device hostname
+3. Replace old device configuration with newly named configuration
+
+## Prerequisite
+1. All devices have valid Central licenses before the workflow is run
+2. Devices are all online and configurable
+3. Devices aren't offline or not able to be configured
+4. Devices do not need to be in the same group
+
+## Installation Steps
+In order to run the script, please complete the steps below:
+1. Clone this repository and `cd` into the workflow directory:
+    ```bash
+    git clone https://github.hpe.com/hpe/central-python-workflows
+    cd central-python-workflows/renaming_aps
+    ```
+   
+2. Install virtual environment (refer https://docs.python.org/3/library/venv.html). Make sure python version 3 is installed in system.
+    ```bash
+    python -m venv env
+    ```
+
+3. Activate the virtual environment
+    In Mac/Linux:
+    ```bash
+    source env/bin/activate
+    ```
+    In Windows:
+    ```bash
+    env/Scripts/activate.bat
+    ```
+
+4. Install the packages required for the script
+    ```bash
+    python -m pip install -r requirements.txt
+    ```
+
+5. Provide the Central API Gateway Base URL & Access Token in the [central_token.json](central_token.json)
+    ```json
+    {
+        "central_info": {
+            "base_url": "<api-gateway-domain-url>",
+            "token": {
+                "access_token": "<api-gateway-access-token>"
+            }
+        },
+        "ssl_verify": true
+    }
+    ```
+    **Note**
+   - [BaseURLs of Aruba Central Clusters](https://developer.arubanetworks.com/aruba-central/docs/api-oauth-access-token#table-domain-urls-for-api-gateway-access)
+   - [Generating Access token from Central UI](https://developer.arubanetworks.com/aruba-central/docs/api-gateway-creating-application-token)
+   - [Generating Access token using OAuth APIs](https://developer.arubanetworks.com/aruba-central/docs/api-oauth-access-token)
+  
```

Comment:
I would add a step here directing the user to update the csv_file. And provide details on what they need to provide there.